craft_providers/lxd/launcher.py

```diff
@@ -72,48 +105,96 @@ def _create_instance(
     """
     logger.info("Creating new instance from remote")
     logger.debug(
-        "Creating new instance from image %r from remote %r.", image_name, image_remote
+        "Creating new instance from image %r from remote %r", image_name, image_remote
     )
```

Comment:
Should these be moved to line 167, where `instance.launch()` is called?

As it is now, the timing/order of these logs doesn't make sense.